aiogram/dispatcher/dispatcher.py

```diff
@@ -1158,8 +1158,8 @@ async def release_key(self, key, chat_id=None, user_id=None):
             raise RuntimeError('This storage does not provide Leaky Bucket')
 
         if user_id is None and chat_id is None:
-            user_id = types.User.get_current()
-            chat_id = types.Chat.get_current()
+            user_id = types.User.get_current().id
```

Comment:
`get_current()` can return `None`, please check this case

Reply:
Ok, I'll do this.
There is also User.get_current().id in throttle function, I'll make check this too.

Reply:
Also, I think it would be better to split this condition 
`if user_id is None and chat_id is None:`

into two separate conditions in throttle, check_key, release_key functions
@Olegt0rr What do you think?

Reply:
> Also, I think it would be better to split this condition
> `if user_id is None and chat_id is None:`
> 
> into two separate conditions in throttle, check_key, release_key functions
> @Olegt0rr What do you think?

maybe it's a bad idea, i need to think more about it,
if these functions will be called with user_id or chat_id argument

bucket will be managed for user_id == chat_id, can't decide if this behavior is always expected.
It looks like this is the expected behavior. And nothing needs to be changed

Reply:
It's a paired id of bucket, so I think there's no reason to split it.
@JrooTJunior what do you think?